wordpress-mobile/gutenberg-mobile
# Fix UI focus Loop when deleting content and blocks

This PR fixes an issue that makes the editor unusable when deleting blocks, or inserting new blocks, typing quickly. Fix #302 

Steps to check the issue is fixed:
- Start the demo app
- Tap into the first Para block (Hello World!)
- Tap Enter and add a new empty block after
- Write something in the block and hit enter quickly
- Repeat the step above. Be fast!
- The UI should respond to input after a while (when all the input events are processed) and there should be no UI loop.

- The same steps above but deleting blocks should not introduce focus loop.

AztecWrapper PR available here: https://github.com/wordpress-mobile/react-native-aztec/pull/86


Other testing scenario:
- Tap on block, try to edit something, and move the focus by tapping on another block. Block holder toolbar should be moved on the other block without issues (*note that there are already other issues reported that makes the block holder not acting fine, this PR should not introduce other problems though).
